pkg/referrerstore/oras/oras.go

```diff
@@ -157,62 +169,108 @@ func (store *orasStore) ListReferrers(ctx context.Context, subjectReference comm
 	return referrerstore.ListReferrersResult{Referrers: referrers}, nil
 }
 
-func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference common.Reference, digest digest.Digest) ([]byte, error) {
-	registryClient, err := store.createRegistryClient(ctx, subjectReference)
+func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference common.Reference, digest digest.Digest, blobDesc ...oci.Descriptor) ([]byte, error) {
+	var err error
+	repository, err := store.createRepository(ctx, subjectReference)
 	if err != nil {
 		return nil, err
 	}
 
-	ref := fmt.Sprintf("%s@%s", subjectReference.Path, digest)
-	desc, err := oras.Copy(ctx, registryClient, ref, store.localCache, "")
+	// resolve blob descriptor if not provided
+	var resolvedBlobDesc oci.Descriptor
```

Comment:
It was not clear as to why we take a list of blobDesc but only use the first one. Sorry, if I am missing any context on ORAS v2

Reply:
the `blobDesc` parameter is similarly mimicking an optional parameter. I couldn't figure out a better way in Go to do this because I don't Go has that built in functionality?

Reply:
switched to normal parameter